changelogs/fragments/8492.yml

```diff
@@ -0,0 +1,2 @@
+fix:
+- Updated DataSource Management to redirect to Discover as well as display the type of the DataSource ([#8492](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/8492))
```

Comment:
Just FYI, the release notes generator requires the `CHANGELOG` description to be under 100 characters. Not sure if it counts the PR number, but the description itself is at 99 characters.

Reply:
Oh yeah my original description was over 100 characters and when I shortened it the changelog was accepted so I'd assume that the changebot checking the character count would be consistent with the release notes generator. That's just an assumption though.